src/dispatch/__tests__/EnhancedAuthentication.test.ts

```diff
@@ -169,29 +154,19 @@ describe('EnhancedAuthentication tests', () => {
     test('when credential is retrieved from basic auth then next credential is retrieved from localStorage', async () => {
```

Comment:
I think the title for this unit test and some others in this file still reference basic auth flow. 

1. The names of the tests need to be updated. ([Example](https://github.com/aws-observability/aws-rum-web/blob/ac471363f6a323fd1a57d898eb612f30a3dee763/src/dispatch/__tests__/EnhancedAuthentication.test.ts#L84)) 
2. We should remove guest role arn from the mocked web client config when using enhanced auth flow ([Example](https://github.com/aws-observability/aws-rum-web/blob/b3f58ceb130e90f99a2d0857bb661e4b9589a02a/src/dispatch/__tests__/EnhancedAuthentication.test.ts#L75))

Reply:
Good catch, however I don't think fixing these fall within the scope of this change.